modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionValidateTask/testShouldThrowExceptionWhenInvalidTokenDefinitionFile.gradle

```diff
@@ -0,0 +1,22 @@
+import com.liferay.gradle.plugins.workspace.configurator.ClientExtensionProjectConfigurator
+
+task runGradleTest
+
+runGradleTest {
+	doLast {
+		String validateClientExtensionsTaskName = ClientExtensionProjectConfigurator.VALIDATE_CLIENT_EXTENSIONS_TASK_NAME
```

Comment:
No need to create `validateClientExtensionsTaskName`, it's single usage, inline it